cmd/vic-machine/create/create.go

```diff
@@ -1091,7 +1091,12 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(c.certPath, 0700)
+	files, err := ioutil.ReadDir(c.certPath)
+	if len(files) > 0 {
```

Comment:
Makes sure it's empty before we go writing files there all willy-nilly. Do note that it tries to load certificates first, so we only bother to do this if the load failed. So this way, vic-machine will not overwrite certificates that were inadvertently used for the install that don't work for the VCH.

The old way was: try to load. If failure -> generate certs. If certs exist and have the same name as the outputs, they get overwritten. This is what was happening to me trying to provide a cert and produced an undue amount of confusion because I didn't realize my certs had been overwritten and couldn't figure out why vic-machine wasn't accepting them :(

I can write a test for this behavior if it makes it clearer and/or makes you feel more comfortable with this functionality